pkg/agent/openflow/client.go

```diff
@@ -1406,6 +1426,17 @@ func (c *client) SendIGMPRemoteReportPacketOut(
 	return c.bridge.SendPacketOut(packetOutObj)
 }
 
+func (c *client) InstallMulticastForwardInboundGroup(groupID binding.GroupIDType) error {
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+
+	group := c.featureMulticast.multicastForwardInboundGroup(groupID, MulticastRoutingTable.GetID())
+	if err := c.ofEntryOperations.AddOFEntries([]binding.OFEntry{group}); err != nil {
+		return fmt.Errorf("error when installing Group %d to forward inbound multicast traffic: %w", groupID, err)
+	}
```

Comment:
Do you forget to save the group into groupCache? How to replay such groups in a re-connect OVS case?

Reply:
updated